src/client/SConscript

```diff
@@ -9,6 +9,7 @@ def scons():
     SConscript('dfs/SConscript')
     SConscript('dfuse/SConscript')
     SConscript('pydaos/SConscript')
+    SConscript('java/SConscript')
```

Comment:
hm, i don't see where java/SConscript is being added. Did you forgot to git add?

Reply:
it's just a preparation. I'll complete it in next PR.

Reply:
ok, and it does not break the build meanwhile to invoke a SConscript that does not exist?

Reply:
sorry. it's failed. I overlooked the email. let me roll it back. thanks